packages/block-library/src/image/edit.native.js

```diff
@@ -71,6 +71,10 @@ const sizeOptions = map( sizeOptionLabels, ( label, option ) => ( { value: optio
 // Default Image ratio 4:3
 const IMAGE_ASPECT_RATIO = 4 / 3;
 
+const getUrlForSlug = ( image, { sizeSlug } ) => {
```

Comment:
Just curious if there was a specific reason you chose to pass an `attributes`-like object with a `sizeSlug` key here instead of just passing the `sizeSlug` directly.

Reply:
Basically just to simplify the call here:
```javascript
 const { image, attributes } = this.props; 	
 const url = getUrlForSlug( image, attributes );
```

But I'm not very happy about it since it also makes this other call a bit more complicated:
```javascript
const url = getUrlForSlug( image, { sizeSlug } );
```